Modeling of use cases

gnitsch opened the pull request:
- cancelled dated vehicle journeys
- late dated vehicle journeys

gnitsch committed:
First draft for Specialisations of Dated Journey Raw Data

- cancelled dated vehicle journeys
- late dated vehicle journeys

Aurige commented on SIRI:
you probably need to rebase here, since we don't want the old submodule back ;-)

Aurige committed:
Update .gitmodules to solve the GML issue

update of the SIRI Git submodule to solve the GML import issue in a more consistent way
To update your local submodule you may need to go to command line and type:
     git submodule sync --recursive
     git submodule update --init --remote

Aurige commented:
I finally did it by hand (updated .gitmodules so it is now identical to the one in master). 
Check it out and update your local submodule you may need to go to command line and type:
     git submodule sync --recursive
     git submodule update --init --remote
